src/pages/Components/Notification/code.md

```diff
@@ -7,26 +7,36 @@ order: 2
 
 ```css
 @import 'settings-tools/_all-settings';
-@import '...';
+@import 'components/_c.notification';
 ```
 
 ## Basic usage
 
-Apply the classes...
+Wrap a h3, p and a a or button tags in a div and apply the `mc-notification` class:
```

Comment:
add the style to highlight HTML tags. like we do here. 
![image](https://user-images.githubusercontent.com/5315842/64543988-7bc16280-d326-11e9-815b-385ccbf91e0e.png)

Wrap a `h3`, `p`, and `a`, ...